src/test/model-viewer-base-spec.ts

```diff
@@ -208,6 +208,46 @@ suite('ModelViewerElementBase', () => {
           expect(dataUrlMatcher.test(element.toDataURL())).to.be.true;
         });
       });
+
+      suite('toBlob', () => {
+        test('produces a blob', async () => {
+          const blob = await element.toBlob();
+          expect(blob).to.not.be.null;
+        });
+
+        test('can convert blob to object URL', async () => {
+          const blob = await element.toBlob();
+          const objectUrl = URL.createObjectURL(blob);
+          const objectUrlMatcher = /^blob\:https?:\//;
+          expect(objectUrlMatcher.test(objectUrl)).to.be.true;
+        });
+
+        test('has size', async () => {
+          const blob = await element.toBlob();
+          expect(blob.size).to.be.greaterThan(0);
+        });
+
+        test('uses fallbacks on unsupported browsers', async () => {
+          // Emulate unsupported browser
+          const restoreCanvasToBlob = spy(HTMLCanvasElement.prototype, 'toBlob', { value: undefined });
+
+          const blob = await element.toBlob();
+          expect(blob).to.not.be.null;
+
+          restoreCanvasToBlob();
+        });
+
+        test('blobs on supported and unsupported browsers are equivalent', async () => {
+          const restoreCanvasToBlob = spy(HTMLCanvasElement.prototype, 'toBlob', { value: undefined });
+          const unsupportedBrowserBlob = await element.toBlob();
+
+          restoreCanvasToBlob();
+
+          const supportedBrowserBlob = await element.toBlob();
+
+          expect(unsupportedBrowserBlob).to.eql(supportedBrowserBlob)
```

Comment:
I think you would probably have to compare the result of `await blob.arrayBuffer()` in order to be sure that the results are equal (unless you are just checking that both results are in fact `Blob`s).

Reply:
Fixed!